DB/src/test/java/io/deephaven/db/tables/utils/TestTableManagementTools.java

```diff
@@ -30,7 +31,8 @@
 import java.util.Arrays;
 import java.util.HashSet;
 
-import static io.deephaven.db.tables.utils.TableTools.col;
+import static io.deephaven.db.tables.utils.TableTools.*;
+import static io.deephaven.db.tables.utils.TableTools.doubleCol;
```

Comment:
The second line here is unused. There are other unused imports we could clean at the same time.